qa/workunits/rbd/rbd_mirror_helpers.sh

```diff
@@ -151,8 +176,10 @@ setup()
     trap cleanup INT TERM EXIT
 
     if [ -n "${RBD_MIRROR_TEMDIR}" ]; then
+	test -d "${RBD_MIRROR_TEMDIR}" ||
 	mkdir "${RBD_MIRROR_TEMDIR}"
 	TEMPDIR="${RBD_MIRROR_TEMDIR}"
+	cd ${TEMPDIR}
```

Comment:
I guess I was asking what these two changes were handling? Is it so that you can re-run the test using an existing cluster / temp directory from a previous test run?